src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -2280,8 +2326,12 @@ namespace HVACVariableRefrigerantFlow {
 				}
 			} else {
 				if ( SameString( cAllCoilTypes( VRFTU( VRFTUNum ).DXCoolCoilType_Num ), cAllCoilTypes( CoilVRF_Cooling ) ) ) {
+					if( VRFTU( VRFTUNum ).TUListIndex > 0 && VRFTU( VRFTUNum ).IndexToTUInTUList > 0 ) {
+						TerminalUnitList( VRFTU( VRFTUNum ).TUListIndex ).CoolingCoilAvailSchPtr( VRFTU( VRFTUNum ).IndexToTUInTUList ) = GetDXCoilAvailSchPtr( DXCoolingCoilType, cAlphaArgs( 12 ), errFlag );
+					} else {
+						VRFTU( VRFTUNum ).CoolingCoilPresent = false;
```

Comment:
Set CoolingCoilPresent to false to avoid issue with NodeID array causing crash later.